packages/opencensus-core/src/tags/propagation/variant-encoding.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const MSB = 0x80;
```

Comment:
Could you add comments for these to explain more clearly what characters they represent and what role they play?

I'm guessing `MSB` is for "most significant bit", and `REST` is for the other bits in a character and that these will be bitwise or'd but wasn't obvious from first glance.